x-pack/plugin/frozen-indices/src/main/java/org/elasticsearch/index/engine/RewriteCachingDirectoryReader.java

```diff
@@ -116,7 +116,7 @@ private RewriteCachingLeafReader(LeafReader original) throws IOException {
                         byte[] maxPackedValue = pointValues.getMaxPackedValue();
                         int numDimensions = pointValues.getNumIndexDimensions();
                         int bytesPerDimension = pointValues.getBytesPerDimension();
-                        int numDataDimensions = pointValues.getNumDataDimensions();
+                        int numDataDimensions = pointValues.getNumDimensions();
```

Comment:
It would be good to rename those variables:

numDimensions -> numIndexDimensions
numDataDimensions -> numDimensions

we can do it in a follow up PR if needed.